.prettierignore

```diff
@@ -7,6 +7,8 @@ packages/tailwind/fonts/font-fallback.js
 
 # This file is automatically generated by TanStack Router.
 routeTree.gen.ts
+# This file is generated as part of the extracting props from the components.
+docgen.ts
```

Comment:
Hva er tanken med å ignore denne her? Jeg la inn i build-scriptet for denne filen at prettier kjøres i etterkant av at den oppdateres:
`"build:docgen": "node build-docs.js && prettier --write docgen.ts"`

Men hvis den ignores her så vil vel ikke det ha noen effekt? Hovedgrunnen til at jeg la det inn var for å rydde opp i JSON-formatteringen som kommer av at vi bruker `JSON.stringify`.

Reply:
Det er fordi jeg får en diff på denne når jeg kjører linting i root av repoet... Ser ikke ut som skrivingen plukker opp riktig config fil kanskje? Eller klarer ikke du gjenskape det samme?

Fjernet det i hvert fall nå. Om det er et problem kan det fikses utenfor denne PRen

Reply:
Aah, ja det er fordi prettier respekterer `gitignore` selvfølgelig. Så prettier blir ikke kjørt på den filen lenger nei. Ser på det 👍

Reply:
PR her: https://github.com/code-obos/grunnmuren/pull/1060